src/components/learner-credit-management/data/hooks/useOfferSummary.js

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react';
```

Comment:
[context] Unchanged. Just moved to standalone file from previous `index.js` file.